.github/workflows/validate_IJ_versions.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        IJ: [IU-2019.3, IU-2020.1, IU-2020.2, IU-2020.3, IU-2021.1]
+        IJ: [IU-2019.3, IU-2020.1, IU-2020.2, IU-2020.3, IU-2021.1, IU-2021.2]
```

Comment:
gradle build should use updated version of gradle-intellij-plugin : https://github.com/JetBrains/gradle-intellij-plugin/releases/tag/v1.1.3